### DISCUSSION TITLE
Does it support 'IN' queries provided by relational systems?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4317] [Q&A] [answered]
greetings.
I want to use 'IN', which is supported by relational DB.
Example > table.column in (1,2,3)

How do I use it for 'filter'?

**Comment by ganigeorgiev** [comment, marked as answer]
The `filter` doesn't have `IN` support at the moment.

As an alternative you can use `||` for similar effect, ex.:
```js
// similar to 'name IN ("test1", "test2")'
name = "test1" || name = "test2"
```

**Reply by jhh78** [reply]
Thank you for answer.

I hope that utility functions will be added as it is updated.

**Reply by HandOfGod94** [reply]
@ganigeorgiev , is this still the case?
If yes, are there plans/roadmap for this feature?
May be I can try picking it up

**Reply by ganigeorgiev** [reply]
Yes, there is no `IN` support at the moment. Feel free to create a PoC but make sure to read the contribution note in the [README](https://github.com/pocketbase/pocketbase?tab=readme-ov-file#contributing).
